src/Tasks/Microsoft.Managed.targets

```diff
@@ -35,6 +35,16 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <DisableTransitiveProjectReferences Condition="'$(BuildProjectReferences)' != 'false' and '$(DisableTransitiveProjectReferences)' == ''">true</DisableTransitiveProjectReferences>
    </PropertyGroup>
 
+   <ItemGroup Condition="'$(IsGraphBuild)' == 'true'">
+      <!-- WPF projects generate a project with a random name at build time and then build the project via the IBuildEngine callbacks.
+           Detect WPF, and exclude the generated project from static graph isolation constraint checking.
```

Comment:
Can you confirm that this works on a `dotnet new wpf` and a template framework XAML-using project?

Reply:
I have both of them in https://github.com/microsoft/MSBuildStaticGraphTester. The msbuild repo does not have a tradition of storing test projects, so I didn't even consider adding them. I could actually add them to the sdk repo, but I have to wait until this change makes it to the sdk.

Reply:
That's all I wanted, no need to add the test here (as part of this; more tests always welcome!)